### PR TITLE
Fixing forum messages 

### PR DESCRIPTION
**Pull request by or313**
@Gal-f 
This PR includes changes due to the fact that the current changes regarding the forumMessages feature in main caused the project not to load.
This changes are mostly to do with links pointing to wrong urls and small view logic fixes.

In addition to that we have added dropdown for choosing a category for a message (New functionality)

All of those changes in one PR so all of the team could fetch this branch onto their workstation and prepare the final demo.

### CODE REVIEW

File: forumMessages/views.py [hunk 1]
```diff
@@ -4,15 +4,20 @@
 
 
 def ViewMessages(request):
+    print("here")
```
Comment (simzacks) [inline forumMessages/views.py]: Unnecessary

File: forumMessages/views.py [hunk 2]
```diff
@@ -4,15 +4,20 @@
...
     messages = Message.objects.order_by('-date')
     if request.method == "POST":
         form = MessageForm(request.POST)
         if form.is_valid():
-            form.save()
-            return redirect(ViewMessages)
+            msg = Message.objects.create(
```
Comment (simzacks) [inline forumMessages/views.py:11]: what happens if form is not valid?

**Review by simzacks — APPROVED** [review]
My 2 comments are not critical issues.
It would be nice to fix them, but not necessary

File: forumMessages/views.py (hunk 1 above)
Comment (Gal-f) [reply forumMessages/views.py]: You're right, it's a redundant line that we used to debug some problems.

File: forumMessages/views.py (hunk 2 above)
Comment (Gal-f) [reply forumMessages/views.py:11]: It's used to trigger cleaning on the fields by Django (input validation).
[See ref here:] (https://docs.djangoproject.com/en/3.1/ref/forms/validation/)
Theoretically, if not valid, nothing will be saved and the user will be redirected nonetheless (`redirect `happens after the if's). Though I don't think it's a possible case for a user to get there, thanks to the input validation.

Comment (simzacks) [reply forumMessages/views.py:11]: makes sense

File: forumMessages/views.py (hunk 1 above)
Comment (Gal-f) [reply forumMessages/views.py]: Fixed it.

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**